packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/TableRows/index.js

```diff
@@ -25,7 +25,7 @@ const TableRows = ({ headers, rows, onOpenModal }) => {
           id: `Settings.permissions.auditLogs.${value}`,
           defaultMessage: getDefaultMessage(value),
         },
-        { model }
+        { model: `(${model})` }
```

Comment:
Could we make these brackets part of the default message? It would be better to pass in plain values so that users are able to localize them to their needs instead of hardcoding them.